src/styles.ts

```diff
@@ -193,7 +193,66 @@ export const styles = css`
 		width: var(--width);
 		min-width: calc(2ch + 0.25em);
 	}
-	:host([type='color']) .line {
+	:host([type='color']) .line,
+	:host([type='checkbox']) .line {
 		display: none;
 	}
+
+	/*Toggle styles */
+
+	.switch {
+		display: block;
+		height: 17px;
+		position: relative;
+		width: 30px;
+	}
+
+	.switch input {
+		display: none;
+	}
```

Comment:
Please follow @megheaiulian's suggestion and create a dedicated style file for the toggle input. Only what is shared should be re-used. `cosmoz-input` does not need the styles for the `cosmoz-toggle`.